### PR TITLE
[Event Hubs Client] Track Two: Third Preview (Documentation and Changelog)

### PR DESCRIPTION
**Pull request by jsquire**
# Summary

The intent of these changes is to update documentation for the preview 3 release.

# Last Upstream Rebase

Friday, September 6, 2019 4:33pm (EDT)

# Related and Follow-Up Issues

- [[Epic] Release Event Hubs Track 2 Library Preview 3 for .Net](https://github.com/Azure/azure-sdk-for-net/issues/7141) (#7141)  
- [Review and update documentation](https://github.com/Azure/azure-sdk-for-net/issues/7183) (#7183)

### CODE REVIEW

**Comment by jsquire** [comment]
//cc: @chrisjbarr

File: sdk/eventhub/Azure.Messaging.EventHubs/README.md [hunk 1]
```diff
@@ -133,26 +133,19 @@ await using (var client = new EventHubClient(connectionString, eventHubName))
 
 To consume events for all partitions of an Event Hub, you'll create an `EventProcessor` for a specific consumer group.  When an Event Hub is created, it provides a default consumer group that can be used to get started.
 
-The `EventProcessor` will delegate processing of events to a `IPartitionProcessor` implementation that you provide, allowing your logic to focus on the logic needed to provide value while the processor holds responsibility for managing the underlying consumer operations.  In our example, we will focus on building the `EventProcessor` and use a very minimal partition processor that does no actual processing.
+The `EventProcessor` will delegate processing of events to a `BasePartitionProcessor` implementation that you provide, allowing your logic to focus on the events receied while the processor holds responsibility for managing the underlying consumer operations.  In our example, we will focus on building the `EventProcessor` and use a very minimal partition processor that does no actual processing.
```
Comment (conniey) [inline sdk/eventhub/Azure.Messaging.EventHubs/README.md]: receied . Good word!

File: sdk/eventhub/Azure.Messaging.EventHubs/README.md [hunk 2]
```diff
@@ -240,12 +241,10 @@ The available samples are:
 - [Consume events from a known position in the Event Hub partition](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample10_ConsumeEventsFromAKnownPosition.cs)  
   An example of consuming events, starting at a well-known position in the Event Hub partition.
   
-- [Consume events from an Event Hub partition in batches](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample11_ConsumeEventsByBatch.cs)
-
+- [Consume events from an Event Hub partition in batches](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventhub/Azure.Messaging.EventHubs/samples/Sample11_ConsumeEventsByBatch.cs)
```
Comment (conniey) [inline sdk/eventhub/Azure.Messaging.EventHubs/README.md]: These links can be made shorter by using relative links (to this README).
`[Consume events from an Event Hub partition in batches](./samples/Sample11_ConsumeEventsByBatch.cs)`

**Review by conniey — APPROVED** [review]

File: sdk/eventhub/Azure.Messaging.EventHubs/README.md (hunk 2 above)
Comment (jsquire) [reply sdk/eventhub/Azure.Messaging.EventHubs/README.md]: I never remember that actually works.   Thanks!   Given that I'm at the end of my day, I'm going to be conservative this time out and just leave them as-is.   I'll revisit when I move back over to the blob storage checkpointererererer skeleton early next week.

**Pull request merged**